### ISSUE TITLE
ParseAssemblyInfo does not work (on .dll and .vb)?

### ISSUE THREAD

**Issue by AnderssonPeter** [cake-build/cake#600]
I have the following script:

```
var webAssemblyFile = File(@"Test.Web\bin\Test.Web.dll");
var webAssembly = ParseAssemblyInfo(webAssemblyFile);
Information("File: " + webAssemblyFile);
Information("AssemblyFileVersion: " + webAssembly.AssemblyFileVersion);
Information("AssemblyInformationalVersion: " + webAssembly.AssemblyInformationalVersion);
Information("AssemblyVersion: " + webAssembly.AssemblyVersion);
Information("Title: " + webAssembly.Title);
Information("File Exists: " + FileExists(webAssemblyFile).ToString());
```

And it generates the following output:

```
File: Test.Web/bin/Test.Web.dll
AssemblyFileVersion: 1.0.0.0
AssemblyInformationalVersion: 1.0.0.0
AssemblyVersion: 1.0.0.0
Title:
File Exists: True
```

Buf if i right click the dll file and use properties i see 2.5.3.0 as version..

i have also tried to point it to my AssemblyInfo.vb and it returns the same result..

```
File: Test.Web/My Project/AssemblyInfo.vb
AssemblyFileVersion: 1.0.0.0
AssemblyInformationalVersion: 1.0.0.0
AssemblyVersion: 1.0.0.0
Title:
File Exists: True
```

So what am i doing wrong?

**Comment by gep13** [comment]
@Petoj87 can you provide a sample of your AssemblyInfo.vb file? Thanks!

**Comment by Petoj87** [comment]
AssemblyInfo.vb:
```
Imports System
Imports System.Reflection
Imports System.Runtime.InteropServices

' General Information about an assembly is controlled through the following 
' set of attributes. Change these attribute values to modify the information
' associated with an assembly.

' Review the values of the assembly attributes
<Assembly: AssemblyTitle("Test.Web")> 
<Assembly: AssemblyDescription("")> 
<Assembly: AssemblyCompany("Test")> 
<Assembly: AssemblyProduct("Test")> 
<Assembly: AssemblyCopyright("Copyright © Test 2012")> 
<Assembly: AssemblyTrademark("")> 

<Assembly: ComVisible(False)> 

'The following GUID is for the ID of the typelib if this project is exposed to COM
<Assembly: Guid("334e9cee-0d47-4d70-924b-b5098a3432cb")>

' Version information for an assembly consists of the following four values:
'
'      Major Version
'      Minor Version 
'      Build Number
'      Revision
'
' You can specify all the values or you can default the Build and Revision Numbers 
' by using the '*' as shown below:
' <Assembly: AssemblyVersion("1.0.*")> 

<Assembly: AssemblyVersion("2.5.3")>
```

**Comment by patriksvensson** [comment]
The AssemblyInfo parser only work with C# at the moment but it's a good idea to add this.

We will probably not support assemblies since this simply is loading an assembly with Assembly.Load and retrieving the values and not within the scope for the AssemblyInfo parser (that parses source code).

**Comment by Petoj87** [comment]
If you ever implement Loading assemblies it would be nice if you did it with `Assembly.ReflectionOnlyLoadFrom` and also do it in a separate app domain that you unload after getting the information that is needed.
But thanks for the tip about just using Assembly it solved my problem..

**Comment by devlead** [comment]
Closed due to inactivity.

**Comment by devlead** [comment]
Fixed by #1380
